[2021-05-17 09:46:27,539] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: matches_links.get_mitre_10_links 2021-04-24T10:00:00+00:00 [queued]>
[2021-05-17 09:46:27,545] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: matches_links.get_mitre_10_links 2021-04-24T10:00:00+00:00 [queued]>
[2021-05-17 09:46:27,546] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-17 09:46:27,546] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2021-05-17 09:46:27,546] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-17 09:46:27,551] {taskinstance.py:1089} INFO - Executing <Task(BashOperator): get_mitre_10_links> on 2021-04-24T10:00:00+00:00
[2021-05-17 09:46:27,553] {standard_task_runner.py:52} INFO - Started process 31868 to run task
[2021-05-17 09:46:27,560] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matches_links', 'get_mitre_10_links', '2021-04-24T10:00:00+00:00', '--job-id', '42', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matches_links/matches_links.py', '--cfg-path', '/var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T/tmpk_5jzlyd', '--error-file', '/var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T/tmpunlvcu2f']
[2021-05-17 09:46:27,561] {standard_task_runner.py:77} INFO - Job 42: Subtask get_mitre_10_links
[2021-05-17 09:46:27,593] {logging_mixin.py:104} INFO - Running <TaskInstance: matches_links.get_mitre_10_links 2021-04-24T10:00:00+00:00 [running]> on host 1.0.0.127.in-addr.arpa
[2021-05-17 09:46:27,616] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=me
AIRFLOW_CTX_DAG_ID=matches_links
AIRFLOW_CTX_TASK_ID=get_mitre_10_links
AIRFLOW_CTX_EXECUTION_DATE=2021-04-24T10:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=backfill__2021-04-24T10:00:00+00:00
[2021-05-17 09:46:27,617] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/tp/ft4rfkmn1kz3dyw8hlfz1dxw0000gn/T
[2021-05-17 09:46:27,617] {bash.py:158} INFO - Running command: Rscript /Users/harry/rugby_data_project_airflow/R/scripts/create_matches_table.R https://www.rugbypass.com/mitre-10-cup/matches/ 2021-04-24
[2021-05-17 09:46:27,625] {bash.py:169} INFO - Output:
[2021-05-17 09:46:28,608] {bash.py:173} INFO - ── Attaching packages ─────────────────────────────────────── tidyverse 1.3.0 ──
[2021-05-17 09:46:28,613] {bash.py:173} INFO - ✔ ggplot2 3.3.3     ✔ purrr   0.3.3
[2021-05-17 09:46:28,613] {bash.py:173} INFO - ✔ tibble  3.0.4     ✔ dplyr   1.0.2
[2021-05-17 09:46:28,613] {bash.py:173} INFO - ✔ tidyr   1.1.2     ✔ stringr 1.4.0
[2021-05-17 09:46:28,614] {bash.py:173} INFO - ✔ readr   1.3.1     ✔ forcats 0.5.0
[2021-05-17 09:46:28,683] {bash.py:173} INFO - ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
[2021-05-17 09:46:28,684] {bash.py:173} INFO - ✖ dplyr::filter() masks stats::filter()
[2021-05-17 09:46:28,684] {bash.py:173} INFO - ✖ dplyr::lag()    masks stats::lag()
[2021-05-17 09:46:28,693] {bash.py:173} INFO - 
[2021-05-17 09:46:28,694] {bash.py:173} INFO - Attaching package: ‘lubridate’
[2021-05-17 09:46:28,694] {bash.py:173} INFO - 
[2021-05-17 09:46:28,694] {bash.py:173} INFO - The following objects are masked from ‘package:base’:
[2021-05-17 09:46:28,695] {bash.py:173} INFO - 
[2021-05-17 09:46:28,695] {bash.py:173} INFO -     date, intersect, setdiff, union
[2021-05-17 09:46:28,695] {bash.py:173} INFO - 
[2021-05-17 09:46:28,696] {bash.py:173} INFO - Loading required package: xml2
[2021-05-17 09:46:28,704] {bash.py:173} INFO - 
[2021-05-17 09:46:28,704] {bash.py:173} INFO - Attaching package: ‘rvest’
[2021-05-17 09:46:28,704] {bash.py:173} INFO - 
[2021-05-17 09:46:28,704] {bash.py:173} INFO - The following object is masked from ‘package:purrr’:
[2021-05-17 09:46:28,705] {bash.py:173} INFO - 
[2021-05-17 09:46:28,705] {bash.py:173} INFO -     pluck
[2021-05-17 09:46:28,705] {bash.py:173} INFO - 
[2021-05-17 09:46:28,706] {bash.py:173} INFO - The following object is masked from ‘package:readr’:
[2021-05-17 09:46:28,706] {bash.py:173} INFO - 
[2021-05-17 09:46:28,706] {bash.py:173} INFO -     guess_encoding
[2021-05-17 09:46:28,706] {bash.py:173} INFO - 
[2021-05-17 09:46:28,707] {bash.py:173} INFO - [1] "2021-04-24"
[2021-05-17 09:46:28,709] {bash.py:173} INFO - [1] "Getting mitre_10_cup links"
[2021-05-17 09:46:30,939] {bash.py:173} INFO - [1] "mitre_10_cup saved"
[2021-05-17 09:46:30,956] {bash.py:177} INFO - Command exited with return code 0
[2021-05-17 09:46:30,967] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=matches_links, task_id=get_mitre_10_links, execution_date=20210424T100000, start_date=20210517T084627, end_date=20210517T084630
[2021-05-17 09:46:30,976] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-17 09:46:31,021] {local_task_job.py:146} INFO - Task exited with return code 0
